app/controllers/hackdays_controller.rb

```diff
@@ -0,0 +1,29 @@
+class HackdaysController < ApplicationController
+
+  def index
+    @hackday = Hackday.new
+    @hackdays = Hackday.all
+  end
+
+  def show
+    @hackday = Hackday.find(params[:id])
```

Comment:
also you should do params[:id].to_i. otherwise its a string and it can be badly optimized.